app/assets/stylesheets/baskets.css.scss

```diff
@@ -41,3 +41,10 @@
     margin-bottom: 0;
   }
 }
+
+.empty {
+  background: $light-gray;
```

Comment:
If you're only setting background-color, you should really use background-color:  instead of background: